`LOAD_METHOD` is still a valid opname but doesn't seem to be fully supported

Porting some code to 3.12 and I get
```
ddtrace/internal/wrapping/__init__.py:520: in wrap
    f.__code__ = code.to_code()
.riot/venv_py3120/lib/python3.12/site-packages/bytecode/bytecode.py:312: in to_code
    bc = self.to_concrete_bytecode(
.riot/venv_py3120/lib/python3.12/site-packages/bytecode/bytecode.py:327: in to_concrete_bytecode
    return converter.to_concrete_bytecode(
.riot/venv_py3120/lib/python3.12/site-packages/bytecode/concrete.py:1401: in to_concrete_bytecode
    self.concrete_instructions()
.riot/venv_py3120/lib/python3.12/site-packages/bytecode/concrete.py:1275: in concrete_instructions
    self.required_caches = c_instr.use_cache_opcodes()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <LOAD_METHOD arg=1 location=InstrLocation(lineno=105, end_lineno=None, col_offset=None, end_col_offset=None)>

    def use_cache_opcodes(self) -> int:
        print(self._opcode)
        return (
            # Not supposed to be used but we need it
>           dis._inline_cache_entries[self._opcode]  # type: ignore
            if sys.version_info >= (3, 11)
            else 0
        )
E       IndexError: list index out of range

.riot/venv_py3120/lib/python3.12/site-packages/bytecode/concrete.py:183: IndexError
```
so it looks like the `LOAD_METHOD` opcode is not supported by 0.15.0. However, `LOAD_ATTR` should provide a workaround by setting the first argument of the tuple to `True`.

--- a/doc/changelog.rst
+++ b/doc/changelog.rst
@@ -6,6 +6,7 @@ ChangeLog
 
 Bugfixes:
 
+- Disallow creating an instruction targeting a pseudo/instrumented opcode PR #133
 - Fixes encoding of 0 as a varint PR #132
 - Correct spelling of "INTRINSIC" in several places; this affected
   some ops in Python 3.12.  PR #131

--- a/src/bytecode/instr.py
+++ b/src/bytecode/instr.py
@@ -14,7 +14,9 @@ except ImportError:
 
 import bytecode as _bytecode
 
-# --- Instruction argument tools and abstractions
+# --- Instruction argument tools and
+
+MIN_INSTRUMENTED_OPCODE = getattr(_opcode, "MIN_INSTRUMENTED_OPCODE", 256)
 
 # Instructions relying on a bit to modify its behavior.
 # The lowest bit is used to encode custom behavior.
@@ -733,6 +735,12 @@ class BaseInstr(Generic[A]):
             opcode = _opcode.opmap[name]
         except KeyError:
             raise ValueError(f"invalid operation name: {name}")
+
+        if opcode >= MIN_INSTRUMENTED_OPCODE:
+            raise ValueError(
+                f"operation {name} is an instrumented or pseudo opcode. "
+                "Only base opcodes are supported"
+            )
 
         self._check_arg(name, opcode, arg)
 

--- a/tests/test_instr.py
+++ b/tests/test_instr.py
@@ -138,6 +138,12 @@ class InstrTests(TestCase):
         self.assertIn("arg", r)
         self.assertIn("_x_", r)
 
+    def test_reject_pseudo_opcode(self):
+        if sys.version_info >= (3, 12):
+            with self.assertRaises(ValueError) as e:
+                Instr("LOAD_METHOD", "x")
+            self.assertIn("is an instrumented or pseudo opcode", str(e.exception))
+
     def test_invalid_arg(self):
         label = Label()
         block = BasicBlock()

